saltstack/optional/pillar/oracle-java/init.sls

```diff
@@ -1 +1,6 @@
-JAVA_HOME: {{ salt['environ.get']('PREINSTALLED_JAVA_HOME') | default('/usr/java/default', True) }}
+JAVA_HOME:
+{% if grains['os_family'] == 'RedHat' %}
+  {{ salt['environ.get']('PREINSTALLED_JAVA_HOME') | default('/usr/java/default', True) }}
+{% else %}
+  /usr/lib/jvm/java-8-oracle
```

Comment:
Is this valid for SUSE too?

Reply:
No, I have not tested it on SUSE.

Reply:
when we implemented oracle jdk support it was only for  Centos/Redhat 7. so when you select oracle optional state then it will fail anyway on Suse

Reply:
Yes, I've talked with Gabor.